CMakeLists.txt

```diff
@@ -103,8 +108,8 @@ add_subdirectory(conversions)
 # Configure files
 ################################################################################
 
-configure_file(cmake/${PROJECT_NAME}ConfigVersion.cmake.in
-    "${PROJECT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake" @ONLY)
```

Comment:
Why do you take all these lines out?

Reply:
I had done this for debugging purposes. I will add them now.